stable/enterprise/values.yaml

```diff
@@ -1061,7 +1061,7 @@ rbacManager:
     annotations: {}
     labels: {}
     nodePort: ""
-    apiVersion: v2
+    apiVersion: v1
```

Comment:
we should update the feeds values.yaml to be v1 as well.

Reply:
done